modules/cache/cache_redis.go

```diff
@@ -12,7 +12,7 @@ import (
 	"code.gitea.io/gitea/modules/nosql"
 
 	"gitea.com/go-chi/cache"
-	"github.com/go-redis/redis/v8"
+	"github.com/redis/go-redis/v9"
```

Comment:
Maybe you mean v9 not v7?

Reply:
@lunny v9 of the library supports v7 of redis